File: Atributos de validación Data Annotations multiidioma/[VB]-Atributos de validación Data Annotations multiidioma/VB/LocalizeDataAnnotations/Models/PersonaResource.vb
Imports System.ComponentModel.DataAnnotations
Imports LocalizeDataAnnotations.My.Resources

Public Class PersonaResource
    Implements IPersona

    <Required(ErrorMessageResourceName:="NombreObligatorio", ErrorMessageResourceType:=GetType(Textos))> _
    <Display(Name:="Nombre", ResourceType:=GetType(Textos))> _
    Public Property Nombre As String Implements IPersona.Nombre

    <Display(Name:="Apellido", ResourceType:=GetType(Textos))> _
    Public Property Apellido As String Implements IPersona.Apellido

    <Display(Name:="Ciudad", ResourceType:=GetType(Textos))> _
    <StringLength(20, MinimumLength:=4, ErrorMessageResourceName:="CiudadErrorLongitud", ErrorMessageResourceType:=GetType(Textos))> _
    Public Property Ciudad As String Implements IPersona.Ciudad

    <Display(Name:="Email", ResourceType:=GetType(Textos))> _
    Public Property Email As String Implements IPersona.Email

End Class

File: Atributos de validación Data Annotations multiidioma/[C#]-Atributos de validación Data Annotations multiidioma/C#/LocalizeDataAnnotations/Models/PersonaResource.cs
using LocalizeDataAnnotations.Content;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Web;

namespace LocalizeDataAnnotations.Models
{
    public class PersonaResource: IPersona
    {

        [Required(ErrorMessageResourceName = "NombreObligatorio", ErrorMessageResourceType = typeof(Textos))]
        [Display(Name="Nombre", ResourceType=typeof(Textos))]
        public string Nombre { get; set; }

        [Display(Name = "Apellido", ResourceType = typeof(Textos))]
        public string Apellido { get; set; }

        [Display(Name = "Ciudad", ResourceType = typeof(Textos))]
        [StringLength(20, MinimumLength = 4, ErrorMessageResourceName="CiudadErrorLongitud", ErrorMessageResourceType=typeof(Textos))]
        public string Ciudad { get; set; }

        [Display(Name = "Email", ResourceType = typeof(Textos))]
        public string Email { get; set; }

    }
}